src/services/messengerSync/testConsumer.js

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env node
+
+var amqp = require('amqplib/callback_api');
+
+amqp.connect('amqp://192.168.99.100', function(error0, connection) {
+    if (error0) {
+        throw error0;
+    }
+    connection.createChannel(function(error1, channel) {
+        if (error1) {
+            throw error1;
+        }
+
+        var queue = 'matrix_sync_populated';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All 'var' declarations must be at the top of the function scope.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5203703)